src/amo/components/SearchForm.scss

```diff
@@ -10,6 +10,6 @@
 .SearchInput-input,
 .SearchForm-form {
   color: $base-color;
-  font-size: $font-size-m;
+  font-size: 14px;
```

Comment:
Do we have a pixel width guide for desktop yet that outlines all the sizes we should be using? That was really handy on mobile.